etc/fixtures/update-genomic-units-fixtures.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+# ./update-genomic-units-fixtures.sh -d <docker-container> -c <mongo_connection_string> -m <mongo_database>
+
+usage() {
+    echo " "
+    echo "usage: $0"
+    echo " "
+    echo " -d docker container name"
+    echo "    Find with 'docker ps' command"
+    echo " -c MongoDB connection string"
+    echo "    (default) localhost:27017"
+    echo " -m Mongo database name"
+    echo "    (default) rosalution_db"
+    echo " "
+    echo " -h Prints usage"
+    echo " "
+    echo "Automatically updates the genomic_units.json fixtures for the initial mongo seed"
+    echo "when deploying a fresh version of Rosalution."
+    echo " "
+    echo "Please install jq for this script to work. https://stedolan.github.io/jq/"
```

Comment:
https://github.com/uab-cgds-worthey/rosalution/blob/787b4880ae5b4af827fcac1669a6c494ffbddff4/etc/api/annotate-all-existing-analyses.sh#L31 The script checks if jq is an available application.  Since jq is required for this script, the same heck is applicable.

Reply:
Added the jq check to the script